line-bot-spring-boot/src/test/java/com/linecorp/bot/spring/boot/integration/destination_handler/IntegrationTestWithDestinationHandler.java

```diff
@@ -115,15 +132,24 @@ public void validCallbackTest() throws Exception {
                .andDo(print())
                .andExpect(status().isOk());
 
+        // ReplyByReturnValueConsumer.reply sends request to messaging api, and doesn't wait the response.
+        // In this test case, we should wait until wiremock receive the request.
+        int maxRetries = 100;
```

Comment:
What about using [Awaitility](https://github.com/awaitility/awaitility#readme)?
```
await().atMost(5, SECONDS).untilAsserted(() -> assertThat(getAllServeEvents().isEmpty())
```

Reply:
It looks good tool but it may be overkill for this one point :p

Reply:
ok, let's reconsider later when we meet same case more